packages-exp/firebase-exp/rollup.config.release.js

```diff
@@ -117,6 +117,43 @@ const appBuilds = [
   }
 ];
 
+const messagingBuilds = [
```

Comment:
It seems like the only difference between messaging and the generic component build is using index.cdn.ts for the input to the UMD build instead of index.ts, is that correct? If that's the case, I wonder if it might be better to put a conditional inside the component build just for messaging. Maybe just hardcode `if (componentName === 'messaging)` and we can make it more general in the future if some other components need special treatment.

Then just make sure to add messaging to the components array in firebase-exp/package.json

Up to you though.